utils/sparse.py

```diff
@@ -142,4 +148,13 @@ def reset_best(self, epoch):
         qat_start = math.floor(max([mod.start_epoch for mod in self.manager.quantization_modifiers])) \
             if self.manager.quantization_modifiers else -1
 
-        return (pruning_start <= epoch <= pruning_end) or epoch == qat_start
\ No newline at end of file
+        return (pruning_start <= epoch <= pruning_end) or epoch == qat_start
+
+    def _apply_one_shot(self):
+        if self.manager is not None:
+            self.manager.apply(self.model)
+            _LOGGER.info(f"Applied recipe {self.train_recipe} in One-Shot")
+        else:
+            _LOGGER.info(f"Recipe not found for one-shot. Got recipe: "
```

Comment:
```suggestion
            _LOGGER.info(f"Training recipe for one-shot application not recognized by the manager. Got recipe: "
```

The info message is not entirely true.